### DISCUSSION TITLE
How long it takes to restart a docker instance if the total jobs are 10Million+ ?

### DISCUSSION THREAD

**Discussion by geminigeek** [cloudamqp/lavinmq#882] [Q&A]
hi,

can anyone suggest how long it takes for the docker instance to restart if i had over 10Million jobs listed when i shutdown the docker container, rather i would like to know what happens when the server is restarted, with many durable jobs on disk?

i waited 15 min the instance is still not accessible ? 15 min is too much for whatever process it takes to load data from disk to memory/queue system?

docker stats showing 100% cpu usage,  htop showing no significant cpu usage ? total disk files on docker volumes are 24 GB , there seems to be no significant disk IO activity also, my system have 128GB ram

using latest 2.0.2

**Comment by spuun** [comment]
When I try locally on my laptop (MBP m1), no docker involved, it loads 25M messages (25GB) in ~45 seconds. It feels like your problem is docker related. If you try with log level info, does it log any progress while loading? It should log lines like `Loaded 512/3124 segments, 4098048 messages`.

We know the startup times aren't perfect and will work on improving this even more.

**Comment by geminigeek** [comment]
hi,

i enabled logs and i am getting these logs on queues which have large amount of jobs 


error kind of a log, these are few last line of logs and then no more logs, and container not responding , some good logs are also there check below
```
<container_name_redacted> | 2024-12-22T10:06:28.704505Z  WARN lmq.message_store[queue: "<qname redacted>", vhost: "cfc"] Msg file size does not match expected value, moving on to next segment
```

same queue with a good log 

```
<container_name_redacted>| 2024-12-22T10:06:26.279669Z  INFO lmq.message_store[queue: "<qname redacted>", vhost: "cfc"] Loaded 128/1076 segments, 2075892 messages
```
